formbricks/formbricks
# Feat/custom closed survey

## What does this PR do?

1. Have added the functionality to add a custom survey close message

Fixes #457


https://github.com/formbricks/formbricks/assets/47187878/dafe440a-53c2-4314-925f-0a49640e022e

Have also fixed the following alignment issue on production

<img width="718" alt="Screenshot 2023-07-03 at 1 12 35 AM" src="https://github.com/formbricks/formbricks/assets/47187878/10b167f9-2c0b-43c5-bd5c-07c1fd231281">

## Type of change




## How should this be tested?



- Create a new or edit an existing survey
- Under the Response Options tab click on adjust "survey closed" message
- add your desired heading and subheading
- Publish and mark the survey as completed
- Check survey closed screen by visiting the survey url

## Checklist
